tests/org.jboss.tools.examples.ui.bot.test/resources/servers/wildfly-blacklist-test-errors-regexes.json

```diff
@@ -21,11 +18,71 @@
 		".*The import org.jboss.quickstarts.ws.jaxws.samples.retail.profile.DiscountRequest cannot be resolved.*",
 		".*The import org.jboss.quickstarts.ws.jaxws.samples.retail.profile.DiscountResponse cannot be resolved.*",
 		".*The import org.jboss.quickstarts.ws.jaxws.samples.retail.profile.ProfileMgmt cannot be resolved.*",
-		".*Type mismatch: cannot convert from ProfileMgmt to ProfileMgmt.*"
+		".*Type mismatch: cannot convert from ProfileMgmt to ProfileMgmt.*",
```

Comment:
I have imported jaxws-retail in my workspace and previous records are not relevant(ot least on JDK11), can you please also review it and delete it if needed? Thank you!

Reply:
Fixed.